author: Fathi Boudra <fabo@debian.org>

kscd.profile.xml should not be installed.
it's an upstream mistake and is shipped by kdemultimedia/kscd.

--- a/kdelirc/profiles/CMakeLists.txt
+++ b/kdelirc/profiles/CMakeLists.txt
@@ -11,7 +11,6 @@ install( FILES 
     konqueror.profile.xml 
     suspend.profile.xml
     shutdown.profile.xml
-    kscd.profile.xml
     DESTINATION ${DATA_INSTALL_DIR}/profiles)
 
 
